src/components/accordion/styles/CdrAccordion.module.scss

```diff
@@ -1,38 +1,32 @@
-@import '../../../styles/settings/index';
-
-//@import url('@rei/cedar/dist/style/cdr-icon.css');
-@import '../../icon/styles/CdrIcon.module';
```

Comment:
how is accordion accessing icons now? 
in the Cedar repo our components can access these as we load all cedar styles but on a project they would only add accordion and not know that the icon and accordion group were also needed